NewSpring/ApollosProject
# (DEV-3903) Adds Save Button to Prayer List

## DESCRIPTION
<img width="429" alt="Screen Shot 2019-07-12 at 9 17 05 AM" src="https://user-images.githubusercontent.com/2659478/61130929-d99c1000-a485-11e9-9116-9e46f515c93f.png">

### What does this PR do, or why is it needed?

This adds a save/unsave button to the prayer list.

### How do I test this PR?

Go in and save/unsave some stuff.
- Check that the UI behaves properly
- Check that it works via the API `savedPrayers` call.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
